lib/Fmt_ast.ml

```diff
@@ -3525,17 +3563,17 @@ and fmt_tydcl_param c ctx ty =
      here. *)
   match ty.ptyp_attributes with
   | [] | _ :: _ :: _ -> noop
-  | [attr] -> fmt_if_k (is_layout attr) (fmt "@ :@ " $ str attr.attr_name.txt)
+  | [attr] -> fmt_if_k (is_layout attr) (fmt_layout_attr c attr)
```

Comment:
I think (?) this shouldn't be necessary any more. It rewrites something like

```
type 'a [@immediate] t
```

to

```
type ('a : immediate) t
```

But we've stopped accepting the previous syntax for a little while. So I think this extra logic can be removed.